package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "dependencies": {
     "@turf/helpers": "6.1.4",
+    "@types/table": "^4.0.6",
```

Comment:
Do we require this in main dependencies? I think only mention in devDependencies is sufficient

Reply:
yes, agreed. Its not supposed to be there. thank you